# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/gpu_cache.rs
# Commit: 96da9d241051
# Full Hash: 96da9d241051d222bdd693bdb519d058a013b59c
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-23 09:46:24
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 8cefa694f811 -r 96da9d241051 gfx/wr/webrender/src/gpu_cache.rs
--- a/gfx/wr/webrender/src/gpu_cache.rs	Fri Mar 22 18:28:31 2019 +0000
+++ b/gfx/wr/webrender/src/gpu_cache.rs	Fri Mar 22 18:28:42 2019 +0000
@@ -714,7 +714,7 @@
     #[cfg(test)]
     pub fn new_for_testing() -> Self {
         let mut cache = Self::new();
-        let mut now = FrameStamp::first(DocumentId(IdNamespace(1), 1));
+        let mut now = FrameStamp::first(DocumentId::new(IdNamespace(1), 1));
         now.advance();
         cache.begin_frame(now);
         cache